### DISCUSSION TITLE
Footer row merge and background color

### DISCUSSION THREAD

**Discussion by MrFahrenheit-55** [bosskmk/pluto_grid#703] [Q&A] [answered]
is it possible to merge cells in a footer row to add a new row to the table? 
I am looking to do something similar to this:
![image](https://user-images.githubusercontent.com/83400730/209574176-7b468dfe-f2dd-4a59-86db-e86075e98983.png)

Also, can you set the background colour in the footer?

Thanks

**Comment by MrFahrenheit-55** [comment, marked as answer]
Resolved

**Reply by darkomenx** [reply]
Perhaps sharing your resolve solution for any others developers.
